### PR TITLE
test: interact with UP instead of KM for LSP20 tests

### PR DESCRIPTION
**Pull request by YamenMerhi**
## What does this PR introduce ?

- Interact with UP instead of KM for LSP20 tests

### CODE REVIEW

File: tests/LSP20CallVerification/LSP6KeyManager/tests/PermissionSetData.test.ts
Comment (CJ42) [inline tests/LSP20CallVerification/LSP6KeyManager/tests/PermissionSetData.test.ts:1]: hmmm the issue around having to hardcode the GAS_LIMIT is still present even via LSP20?
We might need to investigate further

**Review by CJ42 — APPROVED** [review]
LGTM

File: tests/LSP20CallVerification/LSP6KeyManager/tests/PermissionSetData.test.ts
Comment (YamenMerhi) [reply tests/LSP20CallVerification/LSP6KeyManager/tests/PermissionSetData.test.ts:1]: Didn't try to remove it, we can try in the investigation ticket soon

Comment (CJ42) [reply tests/LSP20CallVerification/LSP6KeyManager/tests/PermissionSetData.test.ts:1]: I will add that the issue occurs in LSP20 too, with a link to this comment.

**Review by b00ste — APPROVED** [review]

**Pull request merged**